release-notes/aws/v9.1.1.md

```diff
@@ -0,0 +1,18 @@
+## :zap: Giant Swarm Release 9.1.1 for AWS is now active for you! :zap:
+
+This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also added metrics Service for prometheus-operator support. This enables scraping of metrics and better monitoring on the app.
```

Comment:
One of the sentences reads a bit weird for me, here's a suggestion.

```suggestion
This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also enabled the metrics Service for prometheus-operator. This allows us to scrape for metrics and improves our monitoring the app.
```

Reply:
I've done some wordsmithing, PTAL